src/main/java/org/isf/utils/image/ImageUtil.java

```diff
@@ -61,10 +63,35 @@ public static BufferedImage scaleImage(final BufferedImage src,
         final Graphics2D g2 = resizedImg.createGraphics();
         g2.setBackground(Color.WHITE);
         g2.clearRect(0,0,newWidth, newHeight);
-        g2.drawImage(src, 0, 0, newWidth, newHeight, null);
+        g2.drawImage(bi, 0, 0, newWidth, newHeight, null);
         g2.dispose();
         return resizedImg;
     }
+    
+    /**
+     * Converts a given Image into a BufferedImage
+     *
+     * @param img The Image to be converted
+     * @return The converted BufferedImage
+     */
+    public static BufferedImage toBufferedImage(Image img)
+    {
+        if (img instanceof BufferedImage)
+        {
```

Comment:
Typically (and in the other examples in this PR) the brace is on the same line as the **if**.  Doing so matches the project's formatting standards.  Thanks

Reply:
I corrected as you suggested.